configuration/src/keyVaultClient.d.ts

```diff
@@ -0,0 +1,72 @@
+declare module 'azure-keyvault' {
```

Comment:
The version currently published on npm doesn't. Their next version (currently being worked on in github) will bring in a lot of changes and will include typings, so for now, I just took the simplest approach.